cachemanager/handler_redis.go

```diff
@@ -0,0 +1,101 @@
+package cachemanager
+
+import (
+	"time"
+
+	"gopkg.in/redis.v3"
+	"strconv"
+)
+
+// NewHandlerRedis creates a redis handler
+func NewHandlerRedis(redisConfig *redis.Options, prefix string, ttl time.Duration) Handler {
+	return &handlerRedis{
+		client: redis.NewClient(redisConfig),
+		handler: handler{
+			prefix: prefix,
+			ttl:    ttl,
+		},
+	}
+}
+
+func NewHandlerRedisFromConfiguration(conf *ConfigrationRedis) Handler {
+	return NewHandlerRedis(&redis.Options{
+		Addr:         conf.Addr,
+	}, conf.Prefix, conf.TTL)
+}
+
+type handlerRedis struct {
+	client *redis.Client
+	handler
+}
+
+func (h handlerRedis) Del(key string) error {
+	return h.client.Del(h.buildKey(key)).Err()
+}
+
+func (h handlerRedis) Get(key string) (interface{}, error) {
+	// Initialize
+	var o interface{}
+
+	// Get item
+	i, e := h.client.Get(h.buildKey(key)).Result()
+	if e != nil && e == redis.Nil {
+		return o, ErrCacheMiss
+	}
+
+	return []byte(i), e
+}
+
+func (h handlerRedis) Set(key string, value interface{}, ttl time.Duration) error {
+	// Initialize
+	var v []byte
+
+	// Check value is a slice of bytes
```

Comment:
In redis' case, I think you don't need to transform the value into `[]byte`. Therefore I would remove this part and only return `.client.Set(h.buildKey(key), v, ttl).Err()`.